maven_lambda/metadata.py

```diff
@@ -278,8 +214,6 @@ def get_latest_version(versions_per_path, exclude_snapshots=False):
                 '"{}" does not contain a valid version. See root error.'.format(path)
             ) from error
 
-    if exclude_snapshots:
-        maven_versions = [version for version in maven_versions if not version.is_snapshot]
```

Comment:
@JohanLorenzo This reminded me we need to cleanup mozilla-version too. Should we do that or keep the logic there just in case we'll need it in the future for other projects? I'm tempted to clean that code too to avoid dead code. We could always resurrect should we need to. What do you think?

Reply:
I'd prefer to keep the snapshot logic in mozilla-version, for the sake of being able to process historical data. I've had to support version schemes (https://github.com/mozilla-releng/mozilla-version/pull/59) that were shipped more than 10 years ago because mozilla-version couldn't parse what's in product-details. So I think it's better to keep the snapshot logic there.

Maybe we could make it stricter so that it accepts a known subset of snapshots (just like we do with the old version scheme).

What do you think?

Reply:
Ah, good old' historical data, I missed that. Yeah, let's keep it then 👍

Reply:
Tracked in https://bugzilla.mozilla.org/show_bug.cgi?id=1639133